isis/src/base/objs/PvlTranslationTable/PvlTranslationTable.cpp

```diff
@@ -365,97 +372,149 @@ namespace Isis {
    * Returns the input default value from the translation table corresponding
    * to the output name argument.
    *
-   * @param nName The output name to be used to search the translation table.
+   * @param translationGroupName The output name to be used to search the translation table.
    *
    * @return QString The input default value
    *
-   * @throws iException::Programmer
+   * @throws IException::Programmer
```

Comment:
Do we document the exception here with `@throws`, if the exception is actually thrown from `findTranslationGroup`?